pallets/xcm-benchmarks/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "pallet-xcm-benchmarks"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+version = "0.9.27"
```

Comment:
There are some additional lines missing here. You can take a look at other pallets.

Reply:
Not sure what those lines are, looking at the other pallets in the same folder it could be a description at best.

Reply:
https://github.com/paritytech/cumulus/blob/master/pallets/collator-selection/Cargo.toml